lms/static/scripts/frontend_apps/components/URLPicker.tsx

```diff
@@ -47,10 +47,12 @@ export default function URLPicker({
           setError('Please use a URL that starts with "http" or "https"');
         }
       } else if (isYouTubeURL(rawInputValue)) {
+        // TODO If YouTube is enabled, take the user to the right content picker instead of displaying an error
+        //      instructing how to do it manually
```

Comment:
I took the opportunity to add this comment, as we want to eventually take the user automatically to the `YouTubePicker` if a YouTube URL is introduced in the `URLPicker`.